### DISCUSSION TITLE
Session and Identity Locking Condition.

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#18466] [Q&A] [answered]
Session and Identity Lock was applied after 5 failed Teleport login attempts.
```
tsh login --proxy=xxxxxxxxx.com --auth=local --user=sample-user
Enter password for Teleport user sample-user:
Enter your OTP token:
ERROR: too many incorrect attempts, please try again later
```

Are the conditions under which the account lock is applied documented?
Also, is it possible to change lock conditions?

**Comment by webvictim** [comment, marked as answer]
The defaults are in code: https://github.com/gravitational/teleport/blob/0ee91f6c37eb3a8c5bc98b15f857c95902df50b2/lib/defaults/defaults.go#L231-L233

https://github.com/gravitational/teleport/blob/0ee91f6c37eb3a8c5bc98b15f857c95902df50b2/lib/defaults/defaults.go#L239-L241

Essentially if an incorrect password is provided 5 times in a 20 minute period the account will be locked.

These intervals can't be changed outside of code at the moment.

**Reply by mukitaro** [reply]
Hi webvictim

Thank you for providing an answer to my question.
I understood it.
